spharpy/_deprecation.py

```diff
@@ -1,13 +1,16 @@
-from spharpy.samplings import Coordinates
-from spharpy.samplings import SamplingSphere
+from spharpy.samplings import Coordinates, SamplingSphere
 import pyfar as pf
 
 
+def convert_coordinates_to_pyfar(coordinates):
+    coords_type = type(coordinates)
+    if coords_type is not Coordinates:
+        return coordinates
+    return pf.Coordinates(coordinates.x, coordinates.y, coordinates.z)
```

Comment:
```suggestion
    warn(
        "Spharpy coordinates are deprecated and will be removed in version "
        "0.7.0. Use pyfar.Coordinates or spharpy.SamplingSphere instead",
    return pf.Coordinates(coordinates.x, coordinates.y, coordinates.z)
```
This is a good place to add the deprecation warning, it's always called when converting from spharpy coordinates.
Please also add test checking if the deprecation warning is raised to remind is to remove the code once the time for 0.7.0 has come. (see pyfar on examples)
Alternatively, this could happen in the __init__ of spharpy.Coordinates